code/src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import Wine from '../image/Wine.jpg'
+
+export const Header = () => {
```

Comment:
when you only have one component in a file, you could use export default instead. It would be a clear sign to others who read the code that there is only one compoment and that all of it is exported.